create_toc.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+"""Create table of contents with all epics."""
+
+import os
+
+
+def get_toc():
+    """Get table of contents with all epics."""
+    epic_dirs = [d for d in os.listdir(".") if os.path.isdir(d) and d[:1].isdigit()]
+    toc = []
+    for epic_dir in epic_dirs:
+        try:
+            num = int(epic_dir.split("-", 1)[0])
+            if not 0 <= num < 1000:
+                raise ValueError
+        except ValueError:
+            raise RuntimeError(f"Invalid directory name {epic_dir}")
+        spec_path = os.path.join(epic_dir, "technical_specification.md")
+        if not os.path.exists(spec_path):
+            raise RecursionError(f"No technical_specification.md found in {epic_dir}")
+        for line in open(spec_path):
+            if line.startswith("#"):
```

Comment:
```suggestion
            if line.lstrip().startswith("#"):
```
If you're bothering to strip the name below, then you might as well strip any spaces that wind up in front of the line so it doesn't try to read the scope header as the title.

Reply:
Actually the line needs to start with a `#` but I get your point and fixed it in 6cbe3da.